There also is
```
 /var/jenkins/workspace/Kokkos_PR-8626/simd/src/Kokkos_SIMD_Scalar.hpp:36:34: error: missing '#include "impl/Kokkos_Utilities.hpp"'; 'integral_constant' must be declared before it is used
[C++20-Modules-Clang-19]    36 |   static constexpr Kokkos::Impl::integral_constant<Impl::simd_size_t, 1> size{};
[C++20-Modules-Clang-19]       |                                  ^
[C++20-Modules-Clang-19] /var/jenkins/workspace/Kokkos_PR-8626/core/src/impl/Kokkos_Utilities.hpp:22:8: note: declaration here is not visible
[C++20-Modules-Clang-19]    22 | struct integral_constant {
[C++20-Modules-Clang-19]       |        ^
```

--- a/core/src/Kokkos_Core_Impl.cppm
+++ b/core/src/Kokkos_Core_Impl.cppm
@@ -86,6 +86,7 @@ export {
   // miscellaneous
   namespace Impl {
   using ::Kokkos::Impl::FunctorAnalysis;
+  using ::Kokkos::Impl::integral_constant;
   using ::Kokkos::Impl::python_view_type_impl_t;
   using ::Kokkos::Impl::throw_runtime_exception;
   }  // namespace Impl

--- a/simd/src/Kokkos_SIMD_Common.hpp
+++ b/simd/src/Kokkos_SIMD_Common.hpp
@@ -7,6 +7,7 @@
 #include <Kokkos_Macros.hpp>
 #ifdef KOKKOS_ENABLE_EXPERIMENTAL_CXX20_MODULES
 import kokkos.core;
+import kokkos.core_impl;
 #else
 #include <Kokkos_Core.hpp>
 #endif

